### ISSUE TITLE
A question about the use of kTypeMerge in BlobDB.

### ISSUE THREAD

**Issue by wqshr12345** [facebook/rocksdb#12317]
> Note: Please use Issues only for bug reports. For questions, discussions, feature requests, etc. post to dev group: https://groups.google.com/forum/#!forum/rocksdb or https://www.facebook.com/groups/rocksdb.dev

I've observed that after utilizing BlobDB, only data with kTypeValue is offloaded to blob files, while kTypeMerge remains by default within the SST files. Is there any plan for RocksDB to support kTypeMerge within blob files?

**Comment by jowlyzhang** [comment]
Yes, you are right. With BlobDB, merges are inlined in SST files, while only the value type is separated out into blob files. But as compaction goes on, unless there is outstanding snapshots marking the need for separate versions, the merge entry will eventually get merged into its base value and become a kTypeValue entry too. So there doesn't seem to be a strong reason needing this. We currently don't have plans to support this AFAIK, do you have a strong reason for this feature?
